launcher/image/test/test_ssh_manual.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -euxo pipefail
+
+print_usage() {
+    echo "usage: test_ssh_manual.sh [-i imageName] [-p imageProject]"
+    echo "  -i <imageName>: which image name to use for the VM"
+    echo "  -p <imageProject>: which image project to use for the VM"
+    exit 1
+}
+
+run_ssh_test() {
+    BUILD_ID=$(date +%s)
+    HOME_DIR=$(echo ~)
+    VM_NAME="cs-ssh-test-$BUILD_ID"
+    WORKLOAD_IMAGE='us-west1-docker.pkg.dev/confidential-space-images-dev/cs-integ-test-images/basic-test:latest'
+    ZONE="us-central1-a"
+
+    ACCOUNT_NAME=$(gcloud config list account --format "value(core.account)" | tr @. _)
+    PROJECT_NAME=$(gcloud config get-value project)
+
+    # Create a new VM
+    source create_vm.sh -n $VM_NAME -i $IMAGE_NAME -p $IMAGE_PROJECT -m tee-image-reference=$WORKLOAD_IMAGE,tee-container-log-redirect=true,tee-cmd=["newCmd"],tee-env-ALLOWED_OVERRIDE=overridden,enable-osconfig=TRUE -z $ZONE
+
+    # Add an SSH public key to an OS Login profile
+    gcloud compute os-login ssh-keys add --key-file=$HOME_DIR/.ssh/google_compute_engine.pub || true
+
+    echo "Sleeping so settings have time to propagate."
+    sleep 30
+
+    # SSH into VM with script
+    if [[ $IMAGE_NAME == *"debug"* ]]; then
+        if ssh -i ~/.ssh/google_compute_engine -o StrictHostKeyChecking=no $ACCOUNT_NAME@nic0.$VM_NAME.$ZONE.c.$PROJECT_NAME.internal.gcpnode.com "echo 'SSHABLE'; exit" ; then
```

Comment:
Is it possible to not use the internal address?

Reply:
I was following the current BeyondCorp-style SSH guidance. Are you suggesting using direct IP access or some other path? Thanks!

Reply:
Is it possible to use "gcloud compute ssh \<instance name\>" ?

Reply:
Technically yes, but only for old GCP projects. This syntax adheres to the now recommended BeyondCorp-style SSH flow, which new GCP projects require.

Reply:
SG